### ISSUE TITLE
Compilation problem: trouble with MKL path (?)

### ISSUE THREAD

**Issue by moravveji** [ZimmermanGroup/molecularGSM#16]
Dear 

I am trying to build this software for one of our cluster  users. However, the build fails at the configure step. I see errors w.r.t. MKL not being found; however, few lines before that, the MKL_ROOT is spawned to the correct path. 

I have attached my build script and the error log (the full paths are trimmed out).

In this regard, I have few questions:
- Are the list of dependencies complete, or do I need extra modules to load?
- Is there a specific flag/variable to set to help/enforce CMake find the MKL?
- What are the canonical values for the following variables (now commented out)?
         # GSM_ENABLE_QCHEM=1 \
         # GSM_ENABLE_QCHEM_SF=1 \
         # GSM_ENABLE_ORCA=1 \
         # GSM_ENABLE_GAUSSIAN=1 \
         # GSM_ENABLE_MOLPRO=1 \
- Do you have extra suggestions to improve the build script for building?

Kind regards,
Ehsan
 
[logs.tar.gz](https://github.com/ZimmermanGroup/molecularGSM/files/3961359/logs.tar.gz)

**Comment by moravveji** [comment]
Hey; Any updates on this issue?

**Comment by moravveji** [comment]
closing this ticket, because it is totally ignored here!

**Comment by moravveji** [comment]
thanks @WangXinyan940 for commenting on this abandoned ticket ;-)
In fact, I load the intel module, and MKL is automatically set in the $PATH. E.g.
``` bash
module load intel/2018a
echo $MKLROOT
/software/imkl/2018.1.163-iimpi-2018a/mkl
```
I also set `MKL_ROOT` to `MKLROOT` as well, without solving the issue.. So, that was not the root of my issue.

However, I gave up on this, because this ticket was never answered, and the user whom I was helping also never showed up again!

Best regards
